docs/development/README.md

```diff
@@ -140,4 +157,4 @@ make E2E_TEST=TestOperator/SingleDatacenterCluster e2e-test
 ```
 
 ### Resource Requirements
-Multi-cluster tests will be more resource intensive than other tests. The Docker VM on my MacBook Pro is configured with 6 CPUs and  10 GB of memory for these tests. Your mileage may vary on other operating systems/setups.
\ No newline at end of file
+Multi-cluster tests will be more resource intensive than other tests. The Docker VM used to develop and run these tests is configured with 6 CPUs and 10 GB of memory. Your mileage may vary on other operating systems/setups.
```

Comment:
Why remove the Mac? It's the only platform where such VM configuration is needed.

Reply:
I'll reintroduce it without the `my` which made it not generic enough for a documentation.